game_eggs/minecraft/java/neoforge/egg-neo-forge.json

```diff
@@ -0,0 +1,42 @@
+{
+    "_comment": "DO NOT EDIT: FILE GENERATED AUTOMATICALLY BY PTERODACTYL PANEL - PTERODACTYL.IO",
+    "meta": {
+        "version": "PTDL_v2",
+        "update_url": "https://mirror.invalidpanda.dev/neoforge/egg-neo-forge.json"
```

Comment:
Yhis should be `null` even tho it does not do anything yet

Reply:
Done, but why does this feature exists when it does nothing?

Reply:
There is a feature planned for that value.